CodeGeneration/Sources/SyntaxSupport/ExprNodes.swift

```diff
@@ -1341,7 +1353,8 @@ public let EXPR_NODES: [Node] = [
         kind: .token(choices: [.keyword(text: "repeat")])
       ),
       Child(
-        name: "PatternExpr",
+        name: "Pack",
+        deprecatedName: "PatternExpr",
```

Comment:
@hborla I renamed the expression in `PackExpansionExprSyntax` from `patternExpr` to `pack`. Does this make sense to you. `PatternExpr` didn’t make sense to me because the expression after `repeat` isn’t a pattern in the Swift pattern matching sense.